### PR TITLE
GH-1800 MegaChart Toolbar and IE

### PR DESCRIPTION
**Pull request by GordonSmith**
Buttons are too tall.

Fixes GH-1800

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage Please Review

File: src/layout/Toolbar.css [hunk 1]
```diff
@@ -5,17 +5,26 @@
...
-.layout_Toolbar .toolbar-child {
-    float:right;
-    margin:4px;
-}
-.layout_Toolbar .toolbar-title span {
-    position:relative;
     display: inline-block;
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
```
Comment (jbrundage) [inline src/layout/Toolbar.css]: IE9 needs **-ms-transform: translateY(-50%);**

File: src/layout/Toolbar.css [hunk 2]
```diff
@@ -5,17 +5,26 @@
...
     text-overflow: ellipsis;
     white-space: nowrap;
     overflow: hidden;
 }
+.layout_Toolbar .toolbar-title span {
+}
+.layout_Toolbar .toolbar-child {
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
```
Comment (jbrundage) [inline src/layout/Toolbar.css]: IE9 needs -ms-transform: translateY(-50%);

**Comment by jbrundage** [comment]
@GordonSmith Other than the two css comments, looks good

**Pull request merged**